src/cli/configure/processing/typescript.ts

```diff
@@ -64,7 +64,6 @@ const createImportFromExpression = (
           factory.createNamedImports(
             importNames.map((importName) =>
               factory.createImportSpecifier(
-                false,
```

Comment:
this seems suss

Reply:
https://github.com/microsoft/TypeScript/wiki/API-Breaking-Changes#typescript-45